install_config/configuring_authentication.adoc

```diff
@@ -128,81 +128,132 @@ challenges should set this header to a non-empty value.
+    login: false <1>
+    mappingMethod: "claim" <2>
 ...
 ----
-====
 
-When set to the default `*claim*` value, OAuth will fail if the identity is
-mapped to a previously-existing user name. The following table outlines the use
-cases for the available `*mappingMethod*` parameter values:
+<1> When `fasle` true, unauthenticated token requests from web clients (like the web console) are no redirected to a login page backed by this provider..
 
-[cols="2,8"]
-|===
-|Parameter  | Description
+<2> Controls how mappings are established between this provider’s identities and user objects.
+====
 
-|`*claim*` | The default value. Provisions a user with the identity's preferred
-user name. Fails if a user with that user name is already mapped to another
-identity.
+[[mapping-identities-to-users-table]]
+. Set the `login`parameter to `false`.
+
+. Set the `mappingMethod`parameter as needed.
++
+* `claim`. The default value. Provisions a user with the identity's preferred
+user name. OAuth fails if a user with that user name is already mapped to another
+identity. 
 
-|`*lookup*` | Looks up an existing identity, user identity mapping, and user,
+* `lookup`. Looks up an existing identity, user identity mapping, and user,
 but does not automatically provision users or identities. This allows cluster
 administrators to set up identities and users manually, or using an external
 process.
 
-|`*generate*` | Provisions a user with the identity's preferred user name. If a
+* `generate`. Provisions a user with the identity's preferred user name. If a
 user with the preferred user name is already mapped to an existing identity, a
 unique user name is generated. For example, *myuser2*. This method should not be
 used in combination with external processes that require exact matches between
 {product-title} user names and identity provider user names, such as LDAP group
 sync.
 
-|`*add*` | Provisions a user with the identity's preferred user name. If a user
+* `add`. Provisions a user with the identity's preferred user name. If a user
 with that user name already exists, the identity is mapped to the existing user,
 adding to any existing identity mappings for the user. Required when multiple
 identity providers are configured that identify the same set of users and map to
 the same user names.
-|===
+
+
+. Restart the {product-title} services for the changes to take effect:
++
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-master
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-master
+----
+endif::[]
+
+
+[[ConfiguringIdentityProvider]]
+== Configuring Identity Providers
+
+Use one of the following methods to configure and identity provider to use.
+
 
 [[AllowAllPasswordIdentityProvider]]
 
-=== Allow All
-Set *AllowAllPasswordIdentityProvider* in the `*identityProviders*` stanza to
-allow any non-empty user name and password to log in. This is the default
+=== Configuring the Allow All Identity Provider
```

Comment:
changing headers breaks all my anchor links :)

do we want subpages for the different providers?